### ISSUE TITLE
LUA beautifier can't deal with escape brackets

### ISSUE THREAD

**Issue by lschad** [Glavin001/atom-beautify#1790]
# Description

The results of beautification are not what I expect.

# Input Before Beautification

This is what the code looked like before:

```Lua
Stautsbar = {
  Blank = [[Interface\BUTTONS\WHITE8X8.blp]]
}

        local L = 123
local x = 1
```

# Expected Output

The beautified code should have looked like this:

```Lua
Stautsbar = {
  Blank = [[Interface\BUTTONS\WHITE8X8.blp]]
}

local L = 123
local x = 1
```

# Actual Output

The beautified code actually looked like this:

```Lua
Stautsbar = {
  Blank = [[Interface\BUTTONS\WHITE8X8.blp]]
}

        local L = 123
local x = 1

```

# Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

# Debug

Here is a link to the `debug.md` Gist:  https://gist.github.com/babbelnedd/b277aed05f971bd1d6a72a9a88628d8a

# Checklist

I have:
- [x] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [x] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [x] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

**Comment by babbelnedd** [comment]
another example that goes horrible wrong. basically everything after here gets *destroyed*

```Lua
function Split(input, sep)
  local input = tostring(input)
  if sep == nil then
    sep = "%s"
  end
  local t = {}
  local i = 1
  for str in string.gmatch(input, "([^"..sep.."]+)") do
    t[i] = str
    i = i + 1
  end
  return t
end
```

**Comment by babbelnedd** [comment]
Still an issue

**Comment by babbelnedd** [comment]
Still an issue

**Comment by szeck87** [comment]
@hxsf any insight on this?

**Comment by Glavin001** [comment]
For all those experiencing this issue, please subscribe to https://github.com/Glavin001/atom-beautify/pull/2068 and comment if you have any feedback.

**Comment by Glavin001** [comment]
Published to v0.32.2
